packages/repository-modernizer/src/restructure-filters.js

```diff
@@ -208,12 +223,31 @@ function segregateFilterPaths(filterFileContent, filterPaths) {
                 ) {
                     filterPaths.uiAppsFilters.push(line);
                 } else {
-                    filterPaths.uiContentFilters.push(line);
+                    if (
+                        line.includes("<include") ||
+                        (line.includes("<exclude") && line.includes("mode="))
+                    ) {
+                        filterPaths.uiContentFilters.push(
+                            line.substring(0, line.indexOf("mode")) + "/>"
+                        );
+                        uiContentExtraRoot.push(
+                            "    <filter root" +
+                                line.substring(line.indexOf("="))
+                        );
+                    } else {
+                        filterPaths.uiContentFilters.push(line);
+                    }
                     prevState = false;
                 }
             }
         }
     });
+    filterPaths.uiAppsFilters = filterPaths.uiAppsFilters.concat(
```

Comment:
Can you use `push` instead of `concat` (since `push` appends to end of existing array, and `concat` has to create a new array for merging the 2 arrays, it is much slower). Use this instead : 
```
filterPaths.uiAppsFilters.push(...uiAppsExtraRoot);
```

Reply:
done